tools/docker-network/config.feature.docker.json

```diff
@@ -79,6 +79,10 @@
   },
   "manaInitializer": {
     "FaucetAPI": "http://faucet:8080"
+  },
+  "blockIssuer": {
+    "rateSetter": {
+      "mode": "disabled"
```

Comment:
Same as above

Reply:
I think locally we should disable it now, as it causes some problems with EvilSpammer. Later we can re-enable when other issues are resolved and we can focus on fine-tuning the rate-setter.